taskcluster/xpi_taskgraph/transforms/build.py

```diff
@@ -51,6 +52,13 @@ def tasks_from_manifest(config, jobs):
                     "github_clone_secret"
                 ]
                 artifact_prefix = "xpi/build"
+                repo_url = config.graph_config["taskgraph"]["repositories"][
+                    xpi_config["repo-prefix"]
+                ]["default-repository"]
+                if repo_url.startswith("https"):
+                    raise Exception(
+                        f"{xpi_config['manifest_name']} is a private repo but {repo_url} is a public url!"
```

Comment:
I don't understand, should this bit git+ssh instead of https?  if so, the exception should say how to fix this.